### DISCUSSION TITLE
What is under "dev" section "ipadapter mad scientist" ?

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [cubiq/ComfyUI_IPAdapter_plus#470] [Q&A]
What is under "dev" section "ipadapter mad scientist" ? What is difference from Ipadapter Plus Advanced?

**Comment by cubiq** [comment]
it's undocumented at the moment, I use it mostly for testing. It lets you assign the weight at each layer individually. I'll work on the documentation sooner or later
